images/ose-installer-altinfra.yml

```diff
@@ -1,6 +1,10 @@
 content:
   source:
     dockerfile: images/installer-altinfra/Dockerfile.ci
+    modifications:
+      - action: replace
+        match: "ARG OPENSHIFT_INSTALL_CLUSTER_API=''"
+        replacement: "ARG OPENSHIFT_INSTALL_CLUSTER_API='y'"
```

Comment:
Hm this doesn't make sense to me. IIUC this is going to set `OPENSHIFT_INSTALL_CLUSTER_API ==  y`, in which case the installer build script would try to grab the envtest binaries from the internet, which will fail in OSBS because there is no egress. Am I missing something?

https://github.com/openshift/installer/pull/8273 would make it so that we always build capi providers, and OPENSHIFT_INSTALL_CLUSTER_API would be restricted to grabbing the etcd and kas binaries. That envvar would only need to be used for local development. WDYT?

Reply:
Without this we skip the build of the cluster-api providers. Setting the env var shouldn't be a problem due to https://github.com/openshift/installer/pull/8309/commits/2a373d10cc99aae48149a99aaf18ff0b1d2d7cdf. We can remove this once your 8273 PR merges. But we cannot merge 8273 yet because that would add capi providers to the regular installer image.

Reply:
> shouldn't be a problem due to https://github.com/openshift/installer/commit/2a373d10cc99aae48149a99aaf18ff0b1d2d7cdf.

Ack I just figured this out. We can move discussion of #8273 to a different PR. This resolves for me.